taf/tests/test_updater/update_utils.py

```diff
@@ -256,3 +296,31 @@ def _update_expect_error():
                 assert client_repository.path.exists()
             else:
                 assert not client_repository.path.exists()
+
+
+def invalidate_auth_repo(auth_repo):
```

Comment:
A convention that I try to follow is to put all setup helper function in `conftest`. I think that you should be able to use the setup manager and the same code used to create invalid repositories in `test_clone_invalid`.  When creating the setup manager `setup_manager = SetupManager(auth_repo)` pass in on of the referenced repositories.